vita3k/app/src/screen_render.cpp

```diff
@@ -102,7 +102,7 @@ void gl_screen_renderer::render(const HostState &host) {
     glViewport(static_cast<GLint>(host.viewport_pos.x), static_cast<GLint>(host.viewport_pos.y), static_cast<GLsizei>(host.viewport_size.x),
         static_cast<GLsizei>(host.viewport_size.y));
 
-    glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
+    glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT | GL_STENCIL_BUFFER_BIT);
```

Comment:
We don't use stencil in main framebuffer, so it's no need. What is the reason?

Reply:
I had yellow and blue screen on loading ar nosurge game. At some time this change and maybe some more half-random changes replace yellow to black (that, I think, is more like original behavior). Now i rechecked this and see that this change only has no significant effect.
Thank you very much for explaining, I'll revert this commit.